qiskit_experiments/calibration_management/calibrations.py

```diff
@@ -114,11 +118,8 @@ def __init__(
         self._backend_name = backend_name
         self._backend_version = backend_version
 
-        if isinstance(library, list):
-            raise NotImplementedError(
-                "Passing a list of libraries from which to instantiate "
-                "will be supported in future releases."
-            )
+        if library and not isinstance(library, list):
```

Comment:
In the current PR we are forbidding `Calibrations` from using `Call` which is a rather large change. I wonder if that is sufficient to introduce a `CalibrationsV2` and then follow standard deprecation practices for `Calibrations` ? If this is the case then we can also change names like `library` to `libraries`.

Reply:
I don't think we need to do a full deprecation and transition to `CalibrationsV2`. The repo's README warns about breaking API changes and the user base of `Calibrations` is a subset of that of qiskit-experiments. I think it would be sufficient to provide backwards compatibility for `library`. Mainly, I think this would mean accepting both `library` and `libraries` as keyword arguments to `__init__()` and issuing a deprecation warning if `library` is used (and error if both are used). Then in a later version we could remove the `library` keyword argument. Perhaps the `library` property should also be deprecated but I would guess that is little used outside of the repo and could just be changed to `libraries`. For that case, it is currently a breaking change any way (switching from a library to a list of libraries).

The one other thing is serialization -- we could probably just change `library` to `libraries` there. I don't think anyone would be that upset if they can't deserialize old `Calibrations` because I don't many `Calibrations` have been serialized yet (since serialization was added since the last release)?